### ISSUE TITLE
replace tooltip component

### ISSUE THREAD

**Issue by OlivierCamon** [Conserto/GreenTrackR#60]
The current tooltip component contains several errors:
* it cannot be used by keyboard only users
* it's not easy to uncover
One way to fix this would be to use a detail / summary that can describe the screen.
